client/my-sites/checkout/src/components/jetpack-akismet-checkout-sidebar-plan-upsell/index.tsx

```diff
@@ -119,6 +119,13 @@ const useCalculatedDiscounts = () => {
 				isDiscount: true,
 				isIntroductoryOffer: true,
 			} );
+		} else {
+			// If an annual product has a free trial and the biennial does not, show the multi-uear discount instead
+			priceBreakdown.push( {
+				label: __( 'Multi-year discount*' ),
```

Comment:
Why do we show an asterisk here? I see we use it for the intro discounts but there is an accompanying message that correspondences to it

![image](https://github.com/Automattic/wp-calypso/assets/65001528/a475ce56-385c-4512-bea5-c642f6e16f0b)

If we need an explanation for the multi-year discounts, we should add it, otherwise we should remove the astrisk 😄